docs/advanced/Working_with_git.rst

```diff
@@ -0,0 +1,151 @@
+==================
+Working with git
+==================
+
++----------------------------------+-----------------------------------------------------+
+| Git Commands                     |                What it does                         |
++==================================+=====================================================+
+| git **clone**  <repo>            | Used to clone the repo ``git clone <repo> | <name>``|
++----------------------------------+-----------------------------------------------------+
+| git **commit**                   |  **Commit** an applied change on the given branch   |
++----------------------------------+-----------------------------------------------------+
+| git **remote**                   | `track <http://bit.ly/_fo_rk>`_ remote branch       |
```

Comment:
Ah, let me clarify: the issue is that link shorteners obfuscate the URL that is being linked to, and that the link that shortened links point to can change, not a particular problem with bit.ly.